### PR TITLE
Remove unwraps from crates-io lib

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: src/crates-io/lib.rs [hunk 1]
```diff
@@ -40,6 +40,20 @@ pub enum Error {
     TokenMissing,
     Io(io::Error),
     NotFound,
+    JsonEncodeError,
+    JsonDecodeError,
+}
+
+impl From<json::EncoderError> for Error {
+    fn from(_: json::EncoderError) -> Error {
+        Error::JsonEncodeError
```
Comment (alexcrichton) [inline src/crates-io/lib.rs]: Could each variant retain the payload of the underlying error?

**Comment by jespino** [comment]
I was not sure to include or exclude the underlying error, but I think i take the bad decision. Now added the error, thanks! :smile:

**Comment by alexcrichton** [comment]
@bors: r+ a4a44538e593229db19fc4687f9c6d6362d3b70f

Thanks!

**Comment by jespino** [comment]
This is a bit strange, I'm not sure this is a problem of the PR or a temporary problem on bors running tests

**Pull request merged**